### PR TITLE
Introduce general security policy for Metal3-io

### PR DESCRIPTION
**Pull request by Rozzii**

### CODE REVIEW

File: docs/user-guide/src/security_policy.md
Comment (smoshiur1237) [inline docs/user-guide/src/security_policy.md:125]: It would be better to list Security committee members according to their org: Redhat and Ericsson

File: docs/user-guide/src/security_policy.md
Comment (tuminoid) [inline docs/user-guide/src/security_policy.md:16]: within the 1-7 days => within 7 days

File: docs/user-guide/src/security_policy.md
Comment (tuminoid) [inline docs/user-guide/src/security_policy.md:44]: final call sentence is duplicated here

File: docs/user-guide/src/security_policy.md
Comment (tuminoid) [inline docs/user-guide/src/security_policy.md:65]: Fix Lead -> SL

File: docs/user-guide/src/security_policy.md
Comment (tuminoid) [inline docs/user-guide/src/security_policy.md:120]: Ericsson Software Technologies -> Technology

File: docs/user-guide/src/security_policy.md
Comment (Rozzii) [reply docs/user-guide/src/security_policy.md:125]: I have changed it, is it now okay?

**Review by tuminoid — COMMENTED** [review]
This policy closely follows Kubernetes' policy simplified where due, and besides minor corrections I have no further objections at this time.

File: docs/user-guide/src/security_policy.md
Comment (Rozzii) [reply docs/user-guide/src/security_policy.md]: Done.

File: docs/user-guide/src/security_policy.md
Comment (Rozzii) [reply docs/user-guide/src/security_policy.md]: Done.

File: docs/user-guide/src/security_policy.md
Comment (Rozzii) [reply docs/user-guide/src/security_policy.md]: Done.

File: docs/user-guide/src/security_policy.md
Comment (Rozzii) [reply docs/user-guide/src/security_policy.md]: Done.

**Comment by tuminoid** [comment]
lgtm after the fixes

File: docs/user-guide/src/security_policy.md
Comment (smoshiur1237) [reply docs/user-guide/src/security_policy.md:124]: Thanks. Looks good

**Review by kashifest — APPROVED** [review]

File: docs/user-guide/src/security_policy.md
Comment (Rozzii) [reply docs/user-guide/src/security_policy.md]: Thanks, done.

**Comment by elfosardo** [comment]
LGTM thanks!

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Comment by furkatgofurov7** [comment]
Looks like this needs a force push to fix the Netlify issues?

**Pull request merged**
